src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/include/index.test.js

```diff
@@ -242,16 +341,44 @@ describe('convertInclude', () => {
       href: 'idt2',
       platform: 'highweb',
       type: 'include',
+      idt2: {
+        altText: 'image alt text',
+        dimensions: {
+          small: {
+            href: '/idt2/111-222-333-444-555/image/350',
+            width: 700,
+            height: 1864,
+          },
+          medium: {
+            href: '/idt2/111-222-333-444-555/image/470',
+            width: 940,
+            height: 1864,
+          },
+          large: {
+            href: '/idt2/111-222-333-444-555/image/816',
+            width: 1632,
+            height: 1864,
+          },
+        },
+        copyrightHolder: 'Source: BBC',
+        published: 1550229370779,
+      },
     };
     const expected = {
       type: 'include',
       model: {
         href: 'idt2',
-        required: false,
-        tile: 'IDT2 Include',
-        platform: 'highweb',
         type: 'idt2',
         html: null,
+        imageBlock: {
+          alt: 'image alt text',
+          height: 1864,
+          layout: 'responsive',
+          src: 'https://foobar.com/includes/idt2/111-222-333-444-555/image/816',
+          srcset:
+            'https://foobar.com/includes/idt2/111-222-333-444-555/image/470 470w,https://foobar.com/includes/idt2/111-222-333-444-555/image/816 816w',
```

Comment:
Should the small image be in the `srcset`? I can't see `350` in here from: 
```
small: {
            href: '/idt2/111-222-333-444-555/image/350',
            width: 700,
            height: 1864,
          },
```

Reply:
It shouldn't when it's not on amp, @jimjohnsonrollings stated earlier of using bigger sizes for canonical